### ISSUE TITLE
Synchronizing

### ISSUE THREAD

**Issue by MdotGpunkt** [PhilippC/keepass2android#2670]
### Checks

- [X] I have read the FAQ section, searched the open issues, and still think this is a new bug.

### Describe the bug you encountered:

Synchronizing the database with OneDrive when starting the application does not work. This must always be done manually using the "Synchronize database" function.

### What version of Keepass2Android are you using?

1.10pre

### Which version of Android are you on?

13

**Comment by MdotGpunkt** [comment]
The option "synchronize database" after unlocking was selected.

**Comment by BowlesCR** [comment]
#2760 may have improved this... see if the v1.12 update helps

**Comment by PhilippC** [comment]
I agree. Please try with 1.12 and reopen if the issue persists (https://github.com/PhilippC/keepass2android/releases)
